pkg/monitoring/metrics/forklift-controller/migration_metrics.go

```diff
@@ -0,0 +1,110 @@
+package forklift_controller
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Calculate Migrations metrics every 10 seconds
+func RecordMigrationMetrics(c client.Client) {
+	go func() {
+		for {
+			time.Sleep(10 * time.Second)
+
+			// get all migration objects
+			migrations := api.MigrationList{}
+			err := c.List(context.TODO(), &migrations)
+
+			// if error occurs, retry 10 seconds later
+			if err != nil {
+				fmt.Printf("Metrics Migrations list error: %v\n", err)
+				continue
+			}
+
+			// Holding counter vars used to make gauge update "atomic"
+			var succeededRHV, succeededOCP, succeededOVA, succeededVsphere, succeededOpenstack float64
+			var failedRHV, failedOCP, failedOVA, failedVsphere, failedOpenstack float64
+
+			// for all migrations, count # in executing, succeeded, failed, canceled
```

Comment:
only some of these are counted

Reply:
yes, I've removed some, but maybe for migration its worth adding back canceled and running